spec/controllers/create_or_change_user_requests_controller_spec.rb

```diff
@@ -28,7 +28,6 @@ def valid_change_user_request_params
           "email"=>"subject@digital.cabinet-office.gov.uk",
         },
         "action" => "change_user",
-        "tool_role" => "govt_form",
```

Comment:
Yes, it's intentional but I'll put it in a separate commit and add an appropriate comment